reference/stories/enums.log-line-parser.md

```diff
@@ -27,8 +27,10 @@ These are example tasks that fit the log line parser:
 
 - [C#: enums][implementation-csharp] (reference implementation)
 - [Rust: enums][implementation-rust]
+- [Python: enums][implementation-python]
```

Comment:
This is not hugely important, but I think it should be:

```suggestion
- [Python: log-levels][implementation-python]
```

The same goes for the other story references.